### DISCUSSION TITLE
How to implement elastic search without EcommerceFrameworkBundle

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#7719] [Q&A / Support]
Hello,

We need to use elastic search, we got the documents with EcommerceFrameworkBundle but we did not use EcommerceFrameworkBundle. 
So is there any document for ES?. Also I have checked "Advanced Object Search" but it was for Pimcore backend.

Thanks.

**Comment by sandeep-pim** [comment]
Install Elastic Search 6 on your server and follow this bundle - https://github.com/pimcore/advanced-object-search

**Comment by ChrisB9** [comment]
I am using this bundle: https://github.com/ongr-io/ElasticsearchBundle
and have a custom mapping transforming pimcore models to documents. it works really well but you have to code most things yourself
